config/routes.rb

```diff
@@ -1,11 +1,15 @@
 Rails.application.routes.draw do
+  get 'dashboard/index'
+
+  get 'questions/index'
```

Comment:
Yes we can use namespace as api for API endpoints and resources for RESTful routes, i push it in commit `8eac500`